demisto_sdk/commands/validate/validators/RM_validators/RM101_is_image_path_valid.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import re
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Script, Playbook, Pack]
+
+
+class IsImagePathValidValidator(BaseValidator[ContentTypes]):
+    error_code = "RM101"
+    description = "Validate images absolute paths, and prints the suggested path if it's not valid."
+    rationale = ""
+    error_message = "In {filename} detected the following images URLs which are not raw links: {params}"
+    related_field = ""
+    is_auto_fixable = False
+    expected_git_statuses = [GitStatuses.ADDED, GitStatuses.MODIFIED]
+    related_file_type = [RelatedFileType.README]
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message.format(filename=params[0], params=params[1]),
+                content_object=content_item,
+            )
+            for content_item in content_items
+            if (params := self.is_image_path_valid(content_item))
+        ]
+
+    def is_image_path_valid(self, content_item):
+        if invalid_paths := find_invalid_paths(content_item.readme.file_content):
+            handled_errors = []
+            for path in invalid_paths:
+                path = path[2]
+                alternative_path = path.replace("blob", "raw")
+                handled_errors.append((path, alternative_path))
+            return self.format_error_message(handled_errors, content_item)
+        return ""
+
+    def format_error_message(self, handled_errors, content_item):
+        urls = "\n".join(
+            f"{error[0]} suggested URL {error[1]}" for error in handled_errors
+        )
+        filename = "/".join(
+            content_item.readme.file_path.parts[
+                content_item.readme.file_path.parts.index("Packs") :
+            ]
+        )
+        return (filename, urls)
+
+
+def find_invalid_paths(readme_content):
+    return re.findall(
+        r"(\!\[.*?\]|src\=)(\(|\")(https://github.com/demisto/content/blob/.*?)(\)|\")",
+        readme_content,
+        re.IGNORECASE,
+    )
```

Comment:
Let's consider adding this regex to constants (make sure to add the use of the constant to the old validation as well)
Also, let's consider exporting this method to tools as a general function that gets a string and a regex and run findall with re.IGNORECASE on it.

Reply:
done thanks:)